src/commands/BootstrapCommand.js

```diff
@@ -1,13 +1,12 @@
 import FileSystemUtilities from "../FileSystemUtilities";
-import NpmUtilities from "../NpmUtilities";
 import PackageUtilities from "../PackageUtilities";
+import BootstrapMixin from "../mixins/BootstrapMixin";
 import Command from "../Command";
 import async from "async";
-import find from "lodash.find";
 import path from "path";
 import semver from "semver";
 
-export default class BootstrapCommand extends Command {
+export default class BootstrapCommand extends BootstrapMixin(Command) {
```

Comment:
Ah, interesting.  I had taken that post to be React-specific.

Here I'm intending to use mixins only as a workaround for the lack of multiple inheritance, as [suggested on MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes#Mix-ins).  I see what you're saying about implicit dependencies and complexity, though.

I'll see tomorrow about getting this reorganized to eliminate the need for the mixin.